### PR TITLE
TS-4967: Enable header_freq logging to a specific file.

### PR DESCRIPTION
**Pull request by SolidWallOfCode**

### CODE REVIEW

**Comment by atsci** [comment]
FreeBSD build *successful*! See https://ci.trafficserver.apache.org/job/Github-FreeBSD/1005/ for details.

**Comment by atsci** [comment]
Linux build *successful*! See https://ci.trafficserver.apache.org/job/Github-Linux/897/ for details.

File: plugins/experimental/header_freq/header_freq.cc [hunk 1]
```diff
@@ -46,18 +46,15 @@ static std::map<std::string, unsigned int> client_freq;
 static std::map<std::string, unsigned int> origin_freq;
 
 // for traffic_ctl, name is a convenient identifier
-static const char *ctl_tag = plugin_name;
-static const char *ctl_log = "log"; // log all data
+static const char *ctl_tag              = plugin_name;
+static const char CONTROL_MSG_LOG[]     = "log"; // log all data
+static const size_t CONTROL_MSG_LOG_LEN = sizeof(CONTROL_MSG_LOG) - 1;
```
Comment (jpeach) [inline plugins/experimental/header_freq/header_freq.cc]: Might as well move everything into the anonymous namespace?

Comment (SolidWallOfCode) [inline plugins/experimental/header_freq/header_freq.cc]: Sure, why not?

**Comment by SolidWallOfCode** [comment]
Well, beyond that `TSTextLogObjectCreate` doesn't seem to be documented, it seems more work than is useful for a single write. Note that there's no guarantee multiple invocations will all use the same file.

**Comment by atsci** [comment]
FreeBSD build *successful*! See https://ci.trafficserver.apache.org/job/Github-FreeBSD/1006/ for details.

**Comment by atsci** [comment]
Linux build *successful*! See https://ci.trafficserver.apache.org/job/Github-Linux/898/ for details.

**Comment by SolidWallOfCode** [comment]
I communicated with Petar about this as well and he's fine with it.

**Pull request merged**